src/Geosuggest.jsx

```diff
@@ -57,6 +53,27 @@ const Geosuggest = React.createClass({
     }
   },
 
+  componentDidMount: function() {
+    this.setInputValue(this.props.initialValue);
+
+    var googleMap = (google && google.maps) || this.googleMaps;
+
+    if (!googleMap) {
+      console.error('Google map api was not found in the page.');
+    } else {
+      this.googleMaps = googleMap;
+    }
+
+    this.autocompleteService = new googleMap.places.AutocompleteService();
+    this.geocoder = new googleMap.Geocoder();
+  },
+
+  setInputValue: function(value) {
```

Comment:
Please add a DocBlock comment to functions describing what will happen and the params/returns. See other functions.